### DISCUSSION TITLE
Integrazione motionEye

### DISCUSSION THREAD

**Discussion by issuluca** [indomus/forum#375] [HUB: Home Assistant]
Ho letto le guide, ma non vedo come poi integrare motioneye su homeassistant.
Ho già integrato motioneye tramite l'add-on,
ma poi come si fa ad integrarla nella visualizzazione di home assistant?
Quando provo ad inserire ip e porta assegnata con relativi utente e password mi viene negata la connessione...
Non capisco dove sia il problema.

**Comment by KC-inDomus** [comment]
il problema sarà o nell'URL che gli passi o nelle credenziali. o entrambe le cose...
